src/main/kotlin/com/figure/gradle/semver/internal/tasks/CurrentSemverTask.kt

```diff
@@ -15,15 +15,15 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 
 @CacheableTask
-internal abstract class CurrentSemverTask : DefaultTask() {
+abstract class CurrentSemverTask : DefaultTask() {
```

Comment:
Opening these back up from being internal in case anyone needs to target the tasks for some reason. They can do it by type then.